src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -302,10 +384,26 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
           pipelineRef={this.props.pipelineName}
           namespace={selectedNamespace}
         />
-        <PipelineRunsList
+        <Table
```

Comment:
Can we make the change in the `PipelineRunsList` component instead of here? This keeps a cleaner separation between the container logic and the presentation, and also ensures that existing consumers of the `PipelineRuns` component from our npm package continue to maintain consistency with what we're displaying in the dashboard without requiring them to re-write their integration.

Reply:
Done 😸